service/get_invoice_details.js

```diff
@@ -96,7 +96,7 @@ module.exports = ({cache, invoice, network}, cbk) => {
     // LND connection
     lnd: ['validate', ({}, cbk) => {
       try {
-        return cbk(null, lightningDaemon({}));
+        return cbk(null, lightningDaemon({network}));
```

Comment:
This network is the wrong network, it needs to be the invoice network not the chain context network